Samples/iOS-Swift/iOS-SwiftUITests/LaunchUITests.swift

```diff
@@ -20,6 +20,7 @@ class LaunchUITests: XCTestCase {
         super.tearDown()
     }
 
+    @available(iOS 13, *)
```

Comment:
Please add a comment on why iOS 13 and above. Can you also share insight into what makes you believe it's a tooling issue?

Reply:
Well, we know that the crash report process is on track right now. 
We created more unit tests for it that also run in iOS 12.
I cant prove it because I cant run it in an iOS 12 simulator. but according to the logs, I have a feeling that this is tooling.
If we introduce a bug in the crash report, we will catch it running the test for iOS 13 or newer.

Reply:
Then just add: There seems to be a bug in tooling that is hard to reproduce. Therefore we only run this on iOS 13 and above. If we introduce a bug in crash reporting, we should catch it in the versions above.